src/data/causality.graphql

```diff
@@ -0,0 +1,315 @@
+type ArtsyBidder {
+  id: ID!
+  paddleNumber: ID!
+  userId: ID
+}
+
+"an online (Artsy) bidder, or an offline bidder in the auction"
+union Bidder = ArtsyBidder | OfflineBidder
+
+"DateTime is a scalar value that represents an ISO8601 formatted date and time."
+scalar DateTime
+
+"a long is like an int, figure out why it wasn't included here"
+scalar Long
```

Comment:
I think we may need to adjust our schema somehow to make the Long type exist. I manually added this line because sangria-graphql seems to have its [own special scalar type for `Longs`](https://github.com/sangria-graphql/sangria/blob/4ab54d99e9996e72e01e8ecdf2dafa67426c523a/modules/core/src/main/scala/sangria/schema/package.scala#L30-L33)

Reply:
this has been fixed on the causality end in artsy/causality#812